client/src/AddFriend.js

```diff
@@ -76,6 +76,21 @@ class AddFriend extends React.Component {
 
   resetForm () { this.setState({ friendFName: '', friendLName: '', friendEmail: '' }) }
 
+  handleRemove (id) {
+    const newList = this.state.friendList.filter((item) => item.friendId !== id)
+    this.setState({ friendList: newList })
+    axios.put('/api/friendships', ([this.state.userId, id]))
+      .then(res => {
+        if (res.status >= 200 && res.status < 300) {
+          this.setState({ alertType: alertTable.FRIEND_REMOVED })
+          console.log('deleted friend')
+        }
+      })
```

Comment:
If we update friendList state before we actually do the API request, then there's a minor problem where
1. If it fails, then when the user refreshes, their "deleted" friends will show back up

However, this does prevent the more problematic race condition where clicking delete multiple times could lead to lost deletes. So this is fine for now.